lib/src/Uncertainty/Bayesian/GaussianNonLinearCalibration.cxx

```diff
@@ -389,7 +389,15 @@ Point GaussianNonLinearCalibration::run(const Sample & inputObservations,
   LeastSquaresProblem problem(residualFunction);
   algorithm_.setVerbose(true);
   algorithm_.setProblem(problem);
-  algorithm_.setStartingPoint(getCandidate());
+  try
+  {
+    // If the solver is single start, we can use its setStartingPoint method
+    algorithm_.setStartingPoint(getCandidate());
+  }
+  catch (NotYetImplementedException &) // setStartingPoint is not implemented in MultiStart
```

Comment:
Technically it is false. You should write:
// setStartingPoint is not defined for the algorithm
and it should be a NotDefinedException